### PR TITLE
Feat: Add new UI

### PR DESCRIPTION
**Pull request by rubalv96**
# Description of change
This PR is intended to add the new user interface for the Tangle Explorer website.

### IOTA Tangle Explorer changes:
-   Added new UI on pages:
    

	-   Landing
    
	-   Message
    
	-   Indexed
    
	-   Address
    
	-   Decentralized Identifier
    

-   Added light and dark modes
    
-   Added new modal as a help to understand certain concepts
    
-   New content displayed on Address page:
    

	-   Address QR
    
	-   Total received / Total sent fields
    
	-   Transaction history. For each transaction:
 

		-   Amount (incoming or outgoing)
    
		-   Date on which the transaction was confirmed by a milestone
    
		-   State
    
		-   Number of Inputs/Outputs
    

		-   Filter (incoming/outgoing transactions)
    

-   New content displayed on Message page:
    

	-   Indexation Payload
    

		-   New box to toggle between hexadecimal and UTF-8 data
    
		-   Detect invalid messages in UTF-8 and display their data in hexadecimal only
    

	-   Transaction Payload
    

		-   Amount is added to each input
    

	-   Milestone Payload
    

		-   Actions to go to the previous or next milestone
    

-   New features on Landing page:
    

	-   Feed
    

		-   Pause action
    
		-   Filter by multiple payload options
    
		-   The milestone feed column has been merged into the general one.
    

-   The exclusive page of a milestone is deleted. The information of a milestone will be inside the Milestone Payload of a Message.
    
-   New features on Navbar:
    

	-   The search box is always accessible at the top and it is now possible to search for DID related messages.
    
	-   The currency selector is also located in the navbar accessible from any location
    
	-   Dark/ Light mode toggle button
    

-   New features on Footer:
    

	-   Add social networks

## Type of change
- Enhancement (a non-breaking change which adds functionality)

## Change checklist

- [X] My code follows the contribution guidelines for this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: client/src/app/components/identity/IdentityJsonCompare.scss [hunk 1]
```diff
@@ -28,7 +46,7 @@
   }
 
   .key {
-    color: $gray-10;
+    color: var(--did-color);
   }
 }
```
Comment (brancoder) [inline client/src/app/components/identity/IdentityJsonCompare.scss:52]: Remove unused .difference-select rule

File: client/src/app/components/Footer.tsx [hunk 2]
```diff
@@ -46,41 +104,59 @@ class Footer extends Component<FooterProps, FooterState> {
...
-                        <section className="col fill line-breaks">
-                            {this.state.foundationData?.registeredAddress.value.join("\n")}
-                        </section>
-                        <section className="col fill">
-                            <ul>
-                                {this.state.foundationData?.information.map((info, infoIdx) => (
-                                    <li key={infoIdx}>
-                                        {FoundationDataHelper.createValue(info)}
-                                    </li>
+                <section className="footer--content ">
```
Comment (brancoder) [inline client/src/app/components/Footer.tsx:107]: Remove extra space after class name

**Review by brancoder — COMMENTED** [review]
CSS styling review

File: client/src/app/components/Header.scss [hunk 3]
```diff
@@ -186,3 +505,12 @@ header {
     }
   }
 }
```
[the remark this answers is not in the document]
Comment (rubalv96) [reply client/src/app/components/Header.scss:374]: This should be done in the new changes pushed.

File: client/src/app/routes/og/Bundle.tsx [hunk 4]
```diff
@@ -226,7 +226,7 @@ class Bundle extends Currency<RouteComponentProps<BundleRouteProps>, BundleState
                                 </div>
                                 {this.state.status && (
                                     <div className="card margin-t-s">
-                                        <div className="card--content middle row">
+                                        <div className="card--content middle row margin-t-s ">
```
Comment (brancoder) [inline client/src/app/routes/og/Bundle.tsx:229]: Remove extra space after margin-t-s class name.

File: client/src/app/components/identity/IdentityJsonCompare.scss (hunk 1 above)
Comment (evavirseda) [reply client/src/app/components/identity/IdentityJsonCompare.scss:52]: Resolved

File: client/src/app/components/FiatSelector.tsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+ * Component which will display a currency button.
+ */
+class FiatSelector extends Currency<unknown, CurrencyState> {
+    /**
+     * Render the component.
+     * @returns The node to render.
+     */
+    public render(): ReactNode {
+        return (
+            <div className="select-wrapper ">
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/FiatSelector.tsx]: Resolved

File: client/src/index.scss [hunk 6]
```diff
@@ -81,7 +82,7 @@ body {
 
 ::-webkit-scrollbar-thumb {
   border-radius: 0;
-  background-color: $gray-header;
+  background-color: var(--header-color);
 }
 
 :root {
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/index.scss:92]: Resolved

File: client/src/app/routes/og/Bundle.tsx (hunk 4 above)
Comment (evavirseda) [reply client/src/app/routes/og/Bundle.tsx]: Resolved

File: client/src/app/components/Header.scss [hunk 7]
```diff
@@ -1,188 +1,383 @@
...
-      justify-content: center;
-      width: 150px;
       height: 100%;
-      border: 0;
-      border-radius: 0;
-      outline: 0;
-      background: rgba(72, 87, 118, 0.1);
-      color: $white;
+      margin-left: 40px;
+      color: $gray-6;
```
Comment (brancoder) [inline client/src/app/components/Header.scss:105]: Change rule to color: var(--header-color);

File: client/src/app/components/Header.scss [hunk 8]
```diff
@@ -1,188 +1,383 @@
...
+    .utilities--wrapper {
+      .utilities--dropdown {
+        display: flex;
+        font-family: $metropolis;
+        cursor: pointer;
+
+        .label {
+          @include font-size(14px, 21px);
+
+          color: $gray-6;
```
Comment (brancoder) [inline client/src/app/components/Header.scss:120]: Change rule to color: var(--header-color);

**Review by brancoder — COMMENTED** [review]
Header color changes

File: client/src/app/components/Header.scss (hunk 7 above)
Comment (evavirseda) [reply client/src/app/components/Header.scss:105]: Resolved

File: client/src/app/components/Header.scss (hunk 8 above)
Comment (evavirseda) [reply client/src/app/components/Header.scss:120]: Resolved

Comment (brancoder) [reply client/src/app/components/Header.scss]: Changed to color: var(--navbar-color);

File: client/src/app/components/Header.scss (hunk 7 above)
Comment (brancoder) [reply client/src/app/components/Header.scss]: Changed to color: var(--navbar-color);

File: client/src/scss/themes.scss [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+  --qr-color: #{$gray-9};
+  --box-shadow-down: 0 4px 30px rgba(132, 147, 173, 0.1);
+  --header-identity: #{$gray-daylight};
+  --did-color: #{$gray-7};
+  --search-bg-opacity: 0.6;
+  --card-color: #{$gray-6};
+
+  body.darkmode {
+    --body-background: #091326;
+    --body-color: #{$white};
```
Comment (brancoder) [inline client/src/scss/themes.scss:34]: Change --body-color to #fcfcf - Off White. All of the white text in dark mode is off-white (#fcfcfc) by design.

Comment (evavirseda) [reply client/src/scss/themes.scss]: Resolved

**Review by brancoder — COMMENTED** [review]
Color changes

File: client/src/app/routes/Landing.scss [hunk 10]
```diff
@@ -307,36 +465,37 @@
...
                   button {
-                    @include font-size(12px);
+                    @include font-size(14px, 21px);
 
                     padding: 0;
                     border: 0;
                     background: transparent;
-                    color: inherit;
-                    font-weight: 400;
+                    color: $gray-6;
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/routes/Landing.scss]: Resolved

File: client/src/app/routes/Landing.scss [hunk 11]
```diff
@@ -256,48 +208,254 @@
...
+                    }
+                  }
+                }
+
+                .feed-item--label {
+                  display: none;
+
+                  @include tablet-down {
+                    display: block;
+                    color: $gray-6;
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/routes/Landing.scss]: Resolved

File: client/src/app/components/Footer.scss [hunk 12]
```diff
@@ -1,98 +1,187 @@
...
+    width: 100%;
+    margin: 0;
+    padding: 0;
+
+    .social--media__item {
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      padding: 11px 0;
+      color: #f6f8fc;
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/Footer.scss]: Resolved

File: client/src/app/routes/Landing.scss [hunk 13]
```diff
@@ -256,48 +208,254 @@
...
+                justify-content: space-between;
                 margin-bottom: 10px;
+                padding: 0 8px;
+
+                .label {
+                  @include font-size(12px, 18px);
 
-                .card--label {
                   min-width: 90px;
+                  color: $gray-6;
```
Comment (brancoder) [inline client/src/app/routes/Landing.scss:405]: Change color for dark theme from $gray-6 to $gray-5.

File: client/src/scss/section.scss [hunk 14]
```diff
@@ -0,0 +1,109 @@
...
+  }
+
+  .section--data {
+    margin-bottom: 26px;
+
+    .label {
+      @include font-size(12px);
+
+      margin-bottom: 4px;
+      color: $gray-6;
```
Comment (brancoder) [inline client/src/scss/section.scss:32]: Change color for dark theme from $gray-6 to $gray-5.

File: client/src/app/components/Switcher.scss [hunk 15]
```diff
@@ -1,47 +1,68 @@
...
-    background-color: transparent;
-    color: $white;
-    font-family: $metropolis;
-    cursor: pointer;
-
-    &.selected {
-      background-color: $main-green-highlight;
-      font-weight: 600;
+    margin-right: 16px;
+    color: $gray-6;
```
Comment (brancoder) [inline client/src/app/components/Switcher.scss:14]: Change color for dark theme from $gray-6 to $gray-5.

**Review by brancoder — COMMENTED** [review]
Color changes

**Review by brancoder — COMMENTED** [review]
Color changes

File: client/src/app/routes/Landing.scss (hunk 13 above)
Comment (evavirseda) [reply client/src/app/routes/Landing.scss]: Resolved

File: client/src/scss/section.scss (hunk 14 above)
Comment (evavirseda) [reply client/src/scss/section.scss]: Resolved

File: client/src/app/components/DataToggle.scss [hunk 16]
```diff
@@ -0,0 +1,55 @@
...
+    margin-left: 16px;
+
+    @include phone-down {
+      margin-top: 16px;
+    }
+
+    .data-toggle--tab {
+      margin: 0 2px;
+      padding: 2px 12px;
+      color: $gray-7;
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/DataToggle.scss]: Resolved

File: client/src/app/components/chrysalis/TransactionPayload.scss [hunk 17]
```diff
@@ -0,0 +1,78 @@
...
+      @include tablet-down {
+        margin-top: 23px;
+        margin-left: 0;
+      }
+    }
+
+    .card--content {
+      .card--value {
+        margin-left: auto;
+      }
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/chrysalis/TransactionPayload.scss:64]: Resolved

File: client/src/app/App.tsx [hunk 18]
```diff
@@ -285,43 +297,36 @@ class App extends Component<RouteComponentProps<AppRouteProps> & { config: IConf
      * @returns array of tools
      */
     private getTools() {
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/App.tsx]: Resolved

File: client/src/app/routes/Visualizer.tsx [hunk 19]
```diff
@@ -189,7 +188,7 @@ class Visualizer extends Feeds<RouteComponentProps<VisualizerRouteProps>, Visual
     public render(): ReactNode {
         return (
             <div className={
-                classNames("visualizer", { "dark-mode": this.state.darkMode })
+                classNames("visualizer", { "dark-mode": this.props.darkMode })
```
Comment (brancoder) [inline client/src/app/routes/Visualizer.tsx:191]: Leave only "visualizer" class name and remove "dark-mode" from class names as it is not used for switching to Dark mode.

File: client/src/app/routes/Visualizer.scss [hunk 20]
```diff
@@ -22,7 +23,7 @@
     justify-content: stretch;
     height: 70vh;
     overflow: hidden;
-    border: 1px solid $gray-light;
+    border: 1px solid var(--input-border-color);
     border-radius: 6px;
 
     .viva {
```
Comment (brancoder) [inline client/src/app/routes/Visualizer.scss:29]: Replace .viva background rule from #ffffff to appropriate variable on line 35 and remove rule starting on line 51: `&.dark-mode {
    .graph-border {
      .viva {
        background: $gray-midnight;
      }
    }
  }
`

**Review by brancoder — COMMENTED** [review]
Remove dark-mode class from Visualizer.tsx and Visualizer.scss

File: client/src/app/routes/Visualizer.tsx (hunk 19 above)
Comment (evavirseda) [reply client/src/app/routes/Visualizer.tsx]: Resolved

File: client/src/app/routes/Visualizer.scss (hunk 20 above)
Comment (evavirseda) [reply client/src/app/routes/Visualizer.scss:29]: Resolved

File: client/src/app/routes/chrysalis/Addr.tsx [hunk 21]
```diff
@@ -165,267 +114,340 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
...
-                                                            <MessageButton
-                                                                onClick={() => ClipboardHelper.copy(
-                                                                    output.messageId
-                                                                )}
-                                                                buttonType="copy"
-                                                                labelPosition="top"
+
+                                        </div>
+                                        <table className="transaction--table">
+                                            <tr>
```
Comment (brancoder) [inline client/src/app/routes/chrysalis/Addr.tsx:248]: `<tr>` cannot appear as a child of `<table>`. Add a `<tbody>`, `<thead>` or `<tfoot>` to your code to match the DOM tree generated by the browser.

**Review by brancoder — COMMENTED** [review]
Typo fix and table issue

File: client/src/app/routes/chrysalis/Addr.tsx [hunk 22]
```diff
@@ -165,267 +114,340 @@ class Addr extends AsyncComponent<RouteComponentProps<AddrRouteProps>, AddrState
...
+                                                            {" "}(
+                                                            <FiatValue
+                                                                value={
+                                                                    (this.state.balance ?? 0) +
+                                                                    this.state.sent
+                                                                }
+                                                            />)
+                                                        </div>
+                                                    </div>
+                                                    <div className="section--data ">
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/routes/chrysalis/Addr.tsx]: Resolved

File: client/src/app/routes/chrysalis/Addr.tsx (hunk 21 above)
Comment (evavirseda) [reply client/src/app/routes/chrysalis/Addr.tsx]: Resolved

**Review by brancoder — COMMENTED** [review]
Typo fix

File: client/src/app/components/CurrencyButtonState.tsx [hunk 23]
```diff
@@ -1,6 +1,7 @@
+
```
Comment (brancoder) [inline client/src/app/components/CurrencyButtonState.tsx:1]: Remove empty line.

File: client/src/app/components/DataToggleProps.tsx [hunk 24]
```diff
@@ -0,0 +1,32 @@
+
```
Comment (brancoder) [inline client/src/app/components/DataToggleProps.tsx:1]: Remove empty line

Comment (evavirseda) [reply client/src/app/components/DataToggleProps.tsx]: Resolved

File: client/src/app/components/CurrencyButtonState.tsx (hunk 23 above)
Comment (evavirseda) [reply client/src/app/components/CurrencyButtonState.tsx]: Resolved

File: client/src/app/routes/chrysalis/Indexed.tsx [hunk 25]
```diff
@@ -94,158 +97,122 @@ class Indexed extends AsyncComponent<RouteComponentProps<IndexedRouteProps>, Ind
...
-                                                        this.state.advancedMode))}
-                                            />
-                                        </div>
-                                    </div>
-                                </ToolsPanel>
+                                    ))}
                             </div>
+
+                            {this.state.cursor && (
+                                <div className="indexed-actions flex row margin-t-m margin-b-m ">
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/routes/chrysalis/Indexed.tsx]: Resolved

File: client/src/app/routes/chrysalis/Indexed.tsx [hunk 26]
```diff
@@ -94,158 +97,122 @@ class Indexed extends AsyncComponent<RouteComponentProps<IndexedRouteProps>, Ind
...
+                                    <div className="value">
+                                        There are no messages for this index.
+                                    </div>
+                                )}
+                                {this.state.messageIds &&
+                                    this.state.messageIds.length > 0 &&
+                                    this.state.messageIds.map(messageId => (
+                                        <div
+                                            key={messageId}
+                                            className="indexed-message "
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/routes/chrysalis/Indexed.tsx]: Resolved

File: client/src/app/components/CurrencyButtonState.tsx (hunk 23 above)
Comment (evavirseda) [reply client/src/app/components/CurrencyButtonState.tsx]: Resolved

**Review by brancoder — COMMENTED** [review]
Footer typo fix

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 27]
```diff
@@ -1,132 +1,245 @@
...
+                                    <path d="M10 18L16 12L10 6" stroke="#293858" strokeWidth="2" strokeLinecap="round" />
+                                </svg>
+
+                            </button>
+                        </div>
+                    )}
                 </div>
-                <div className="card--content">
-                    <div className="card--label">
+                <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:95]: Remove extra space from className.

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 28]
```diff
@@ -1,132 +1,245 @@
...
+                    <div className="label">
                         Index
                     </div>
-                    <div className="card--value">
+                    <div className="value">
                         {this.props.payload.index}
                     </div>
-                    <div className="card--label">
+                </div>
+                <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:103]: Remove extra space from className.

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 29]
```diff
@@ -1,132 +1,245 @@
...
-                                            labelPosition="top"
-                                        />
-                                    </div>
-                                </React.Fragment>
-                            ))}
-                            <div className="card--label">
+                </div>
+                {this.props.advancedMode && (
+                    <React.Fragment>
+                        <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:117]: Remove extra space from className

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 30]
```diff
@@ -1,132 +1,245 @@
...
+                            <div className="value code">
                                 {this.props.payload.inclusionMerkleProof}
                             </div>
-                            {this.props.payload.nextPoWScore !== 0 && this.props.payload.nextPoWScoreMilestoneIndex !== 0 && (
-                                <React.Fragment>
-                                    <div className="card--label">
+                        </div>
+                        {this.props.payload.nextPoWScore !== 0 && this.props.payload.nextPoWScoreMilestoneIndex !== 0 && (
+                            <React.Fragment>
+                                <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:127]: Remove extra space from className.

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 31]
```diff
@@ -1,132 +1,245 @@
...
+                                    <div className="label">
                                         Next PoW Score
                                     </div>
-                                    <div className="card--value">
+                                    <div className="value code">
                                         {this.props.payload.nextPoWScore}
                                     </div>
-                                    <div className="card--label">
+                                </div>
+                                <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:135]: Remove extra space from className.

File: client/src/app/components/chrysalis/MilestonePayload.tsx [hunk 32]
```diff
@@ -1,132 +1,245 @@
...
+                                <div className="value code">
+                                    {this.props.payload.publicKeys?.map(pubKey => (
+                                        <div key={pubKey} className="margin-b-s">
+                                            {pubKey}
+                                        </div>
+                                    ))}
+                                </div>
+                            </div>
+                        )}
+                        <div className="section--data ">
```
Comment (brancoder) [inline client/src/app/components/chrysalis/MilestonePayload.tsx:160]: Remove extra space from className.

**Review by brancoder — COMMENTED** [review]
MilestonePayload.tsx

File: client/src/app/components/Footer.tsx [hunk 33]
```diff
@@ -46,41 +104,59 @@ class Footer extends Component<FooterProps, FooterState> {
...
                                 ))}
-                            </ul>
-                        </section>
+                        </div>
+                        <hr className="sep" />
+                        <div className="row foundation-data">
+                            <div className="margin-t-m">
+                                <img src={logoFooter} alt="IOTA" />
+                            </div>
+                            <section className=" line-breaks">
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/Footer.tsx]: Resolved

File: client/src/app/components/Footer.tsx [hunk 34]
```diff
@@ -46,41 +104,59 @@ class Footer extends Component<FooterProps, FooterState> {
...
+                                {this.state.foundationData?.registeredAddress.value.join("\n")}
+                            </section>
+                            <section className="">
```
[the remark this answers is not in the document]
Comment (evavirseda) [reply client/src/app/components/Footer.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 27 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 28 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 29 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 30 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 31 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

File: client/src/app/components/chrysalis/MilestonePayload.tsx (hunk 32 above)
Comment (evavirseda) [reply client/src/app/components/chrysalis/MilestonePayload.tsx]: Resolved

**Review by brancoder — APPROVED** [review]

**Comment by laumair** [comment]
### IOTA Tangle Explorer changes:
-   Added new UI on pages:
    

	-   Landing
    
	-   Message
    
	-   Indexed
    
	-   Address
    
	-   Decentralized Identifier
    

-   Added light and dark modes
    
-   Added new modal as a help to understand certain concepts
    
-   New content displayed on Address page:
    

	-   Address QR
    
	-   Total received / Total sent fields
    
	-   Transaction history. For each transaction:
 

		-   Amount (incoming or outgoing)
    
		-   Date on which the transaction was confirmed by a milestone
    
		-   State
    
		-   Number of Inputs/Outputs
    

		-   Filter (incoming/outgoing transactions)
    

-   New content displayed on Message page:
    

	-   Indexation Payload
    

		-   New box to toggle between hexadecimal and UTF-8 data
    
		-   Detect invalid messages in UTF-8 and display their data in hexadecimal only
    

	-   Transaction Payload
    

		-   Amount is added to each input
    

	-   Milestone Payload
    

		-   Actions to go to the previous or next milestone
    

-   New features on Landing page:
    

	-   Feed
    

		-   Pause action
    
		-   Filter by multiple payload options
    
		-   The milestone feed column has been merged into the general one.
    

-   The exclusive page of a milestone is deleted. The information of a milestone will be inside the Milestone Payload of a Message.
    
-   New features on Navbar:
    

	-   The search box is always accessible at the top and it is now possible to search for DID related messages.
    
	-   The currency selector is also located in the navbar accessible from any location
    
	-   Dark/ Light mode toggle button
    

-   New features on Footer:
    

	-   Add social networks

**Pull request merged**